packages/react/package.json

```diff
@@ -145,7 +145,9 @@
     "README.zh.md"
   ],
   "scripts": {
-    "api-extractor": "api-extractor run --verbose"
+    "api-extractor": "api-extractor run --verbose",
+    "build": "rslib build",
```

Comment:
Perhaps we could move `scripts.build` to each sub-workspace (e.g.: `runtime`, `components`, `worklet-runtime`).

Reply:
I have split `worklet-runtime` to its own package.json, `components` is not a package, so I keep it here